CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ User-visible changes worth mentioning.
 
 - [#PR ID] Add your PR description here.
 - [#1477] Respect `ActiveRecord::Base.pluralize_table_names` for Doorkeeper table names.
+- [#1488] Verify client authentication for Resource Owner Password Grant when
```

Comment:
Look like outdated copy :thinking: 

We already released 5.5 https://github.com/doorkeeper-gem/doorkeeper/blob/main/CHANGELOG.md#main

Reply:
@nbulaj whoops indeed, I had an old checkout and missed the rename from `master` to `main`! :facepalm: 

Rebased now, and moved the changelog entry up.